docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,207 @@ The ACS will be used for V&V testing. It is defined as:
 - Includes mail/online, telephone, and personal visits 
 
 
+Taxes
+^^^^^
+
+Taxes, as we all know, can contain many different forms and processes. 
+For this model, we will split the tax information into two main sections: 
+W2/1099 forms from employers; and 1040 forms from simulants. We will look 
+at these separately, starting with W2 and 1099 forms. 
+
+
+W2 and 1099 Forms
+'''''''''''''''''
+
+**When to Sample** 
+
+- Sample compiled on the time step containing Jan 1st of each year (the time step might end on Jan 2nd, Jan 15th, Jan 27th, etc.)
+- However, we will want to track every job a simulant has had for any time step within a calendar year, which might require additional observers. If a simulant changes jobs in March of 2020, their tax documents on Jan 1st of 2021 will need to include both their current job, and their job from February of 2020. 
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - DOB 
+  * - Mailing Address 
+  * - Social Security Number 
+  * - Income 
```

Comment:
sidenote: we need docs describing an income column (we currently agreed on slack that we'll assign the exact same income to everyone who is employed, and zero to everyone else)

Reply:
Yes, I misunderstood. I thought income was randomly assigned, so people would have higher and lower incomes even if the numbers were random. I think income will be built out more in the future, so I want to leave these pieces in the docs, but I will clarify that the income does not apply for now.